runtime/parachains/src/ump.rs

```diff
@@ -148,6 +144,9 @@ impl fmt::Debug for AcceptanceCheckErr {
 pub trait Config: frame_system::Config + configuration::Config {
 	/// A place where all received upward messages are funneled.
 	type UmpSink: UmpSink;
+
+	/// The factor by which the weight limit it multiplied for the first UMP message to execute with.
+	type FirstMessageFactorPercent: Get<Weight>;
```

Comment:
Why not use perbill?

Reply:
No huge reason but didn't see a huge need to pull in another type for a simple parameter.